### PR TITLE
`Development`: Fix minor grammar and spelling issues in README.md

### PR DESCRIPTION
**Pull request by fh314**
#### General

- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.ase.in.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).

### Motivation and Context

- Inconsistent use of American and non-American spelling
- Minor typos/ spelling/ grammar mistakes

### Description
See comparison

### Steps for Testing

Prerequisites:
-

1. Compare the changes

#### Code Review
- [ ] Review 1
- [ ] Review 2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,13 +31,13 @@
...
 7. **Grading**: Instructors can configure grade keys for courses and exams to automatically calculate grades and display them to students. Grades can be easily exported as csv files to upload them into university systems (such as Campus online)
-8. **Assessment**: Artemis uses double-blind grading and structured grading criteria to improve consistency and fairness. It integrates an assessment training process (based on example submissions and example assessments defined by the instructor), has a grading leaderboard and allows students to rated the assessments. Students can complaint or ask for more feedback.   
-9. **Discussion**: Instructors can post announcements. Students can ask questions, post comments and react to other post. Tutors can filter unanswered questions.
+8. **Assessment**: Artemis uses double-blind grading and structured grading criteria to improve consistency and fairness. It integrates an assessment training process (based on example submissions and example assessments defined by the instructor), has a grading leaderboard and allows students to rate the assessments. Students can complain or ask for more feedback.
```
Comment (Malyuk-A) [inline README.md:34]:
```suggestion
8. **Assessment**: Artemis uses double-blind grading and structured grading criteria to improve consistency and fairness. It integrates an assessment training process (based on example submissions and example assessments defined by the instructor), has a grading leader board, and allows students to rate the assessments. Students can complain or ask for more feedback.   
```
image

File: README.md [hunk 2]
```diff
@@ -31,13 +31,13 @@
...
-8. **Assessment**: Artemis uses double-blind grading and structured grading criteria to improve consistency and fairness. It integrates an assessment training process (based on example submissions and example assessments defined by the instructor), has a grading leaderboard and allows students to rated the assessments. Students can complaint or ask for more feedback.   
-9. **Discussion**: Instructors can post announcements. Students can ask questions, post comments and react to other post. Tutors can filter unanswered questions.
+8. **Assessment**: Artemis uses double-blind grading and structured grading criteria to improve consistency and fairness. It integrates an assessment training process (based on example submissions and example assessments defined by the instructor), has a grading leaderboard and allows students to rate the assessments. Students can complain or ask for more feedback.   
+9. **Discussion**: Instructors can post announcements. Students can ask questions, post comments and react to other posts. Tutors can filter unanswered questions.
```
Comment (Malyuk-A) [inline README.md:35]:
```suggestion
9. **Discussion**: Instructors can post announcements. Students can ask questions, post comments, and react to other posts. Tutors can filter unanswered questions.
```

File: README.md [hunk 3]
```diff
@@ -31,13 +31,13 @@
...
 10. **Notifications**: Artemis supports customizable web and email notifications. Users can enable and disable different notification types.
 11. **Team exercises**: Instructors can configure team exercises with real time collaboration and dedicated tutors per team.
 12. **Lectures**: Instructors can upload lecture slides, divide lectures into units, integrate video streams, lecture recordings and exercises into lectures, and define learning goals.
 13. **Integrated markdown editor**: Instructors define problem statements using Markdown to easily format the text, and to integrate images and links.
-14. **Plagiarism checks**: Artemis integrates plagiarism checks for programming exercises (based on [JPlag](https://github.com/jplag/JPlag)), text exercises and modeling exercises. It allows to notify students about identified plagiarism. Students can review and comment on the allegation. 
+14. **Plagiarism checks**: Artemis integrates plagiarism checks for programming exercises (based on [JPlag](https://github.com/jplag/JPlag)), text exercises and modeling exercises. It allows notifying students about identified plagiarism. Students can review and comment on the allegation.
```
Comment (Malyuk-A) [inline README.md:40]:
```suggestion
14. **Plagiarism checks**: Artemis integrates plagiarism checks for programming exercises (based on [JPlag](https://github.com/jplag/JPlag)), text exercises, and modeling exercises. It allows notifying students about identified plagiarism. Students can review and comment on the allegation. 
```

**Review by Malyuk-A — COMMENTED** [review]
Nice changes 👍
I run the changes through a few spell checkers and found a few minor improvements

File: README.md (hunk 3 above)
Comment (fh314) [reply README.md:40]: I wasn't sure about the oxford commas, I think there are a few more places that we would have to set them to keep it consistent.

**Review by willergeorg — APPROVED** [review]
Changes look good in my opinion.

**Review by krusche — APPROVED** [review]

**Comment by krusche** [comment]
Thanks

**Pull request merged**